pom.xml

```diff
@@ -349,6 +349,18 @@
       </plugin>
       <!-- End publish to maven central -->
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.3.0</version>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>dev.openfeature.sdk</Automatic-Module-Name>
```

Comment:
My understanding is that that module name should match the "highest level" package name, so this seems correct.

Reply:
@toddbaert  Thank you for the review!  Do you know when this will get merged?  Is there a public release schedule?

Reply:
Hey @ARobertsCollibra, we can release right after this is merged. A new release will happen automatically after [this PR](https://github.com/open-feature/java-sdk/pull/331) is merged.

Reply:
This change was included in `v1.3.1`. Thanks for the help!